### ISSUE TITLE
Update ArcGIS Pro on HydroVIS to v3.2

### ISSUE THREAD

**Issue by MonicaStone-NOAA** [NOAA-OWP/hydrovis#381]
Update the version of ArcGIS Pro to 3.0 in HydroVIS.  This is necessary for some of the service development being conducted by the Catherines, who need it upgraded on their EC2 instance.  And it is necessary for updating the custom print service.

**Comment by CoreyKrewson-NOAA** [comment]
After talking to ESRI, there is some compatibility issues between pro and arcgis enterprise versions. We need to keep this in mind. We shouldn't upgrade pro until we can upgrade our EGIS stack

https://pro.arcgis.com/en/pro-app/latest/help/analysis/geoprocessing/share-analysis/geoprocessing-service-compatibility.htm

**Comment by nickchadwick-noaa** [comment]
Waiting for mid November for when V3.2 comes out.

**Comment by RobHanna-NOAA** [comment]
There are some EC2's have it installed on them. Are they on the list already?

**Comment by nickchadwick-noaa** [comment]
I believe previously ArcGIS Pro was only manually updated on a couple of EC2s for people who requested it at the time.

I am reopening this ticket so make sure that the new version is baked into the EC2 image and the remaining existing EC2s are either remade or manually upgraded.

**Comment by RobHanna-NOAA** [comment]
Sounds good. If if eveyrone is ok, it might be good to have @EdisonOrellana-NOAA chase down which are all required for upgrades. This include a good handful of ec2, such as the pipeline ec2's, pgadmin ec2's and some individual ones like one for Edison, Lorne, Tyler, etc. Yes.. let's upgrade Tylers as we are starting to take it over.

Also.. we need to look for similar ones in the UAT enviros. 

I believe we are far enough behind on versions, it would be a good idea to get them all found and updated. Appx 2.8 to 3.4 (??)

Talking to different people, will help flush out most of them , and a few we can open up and look at. 

Also.. If everyone agrees @derekgiardino  will need to approve if we want to hunt down and upgrade all or take it piece meal. We do know of a few that we really do the need the upgrade.

REMEMBER FOLKS: Caution on putting actual server names in issues cards, as this is  a public site.

**Comment by RobHanna-NOAA** [comment]
oh.. consistency is great, but I heard that Arc recently and radically increased their license costs. hummm.

**Comment by EdisonOrellana-NOAA** [comment]
Good point, @RobHanna-NOAA  . I haven't done the calculation but I believe the supposition was that Kubernetes ArcGIS licensing costs are cheaper than AWS compute costs for Windows based ArcGIS.

**Comment by RobHanna-NOAA** [comment]
oh ya.. and we have a few ras2fim / FIM dev EC2s that have arc in them as well but just a few. But, I am pretty sure they won't need the upgrades, but TBD. Depends if/how we use parquet in the next versions and if we can use QGIS instead. I am guessing a fair bit of this we might be able to even switch to using QGIS, but that depends on how we continue to publish, work with Arc.  ie) some test / dev might be fine in QGIS, but maybe only arc for mapx and publishing. Just a thought. So maybe even different configurations on different machines based on that need.  Definitely some discussion required.
